GetandTake/Pages/Categories/Category.cshtml.cs

```diff
@@ -7,13 +7,21 @@ namespace GetandTake.Pages.Categories;
 public class CategoryModel : PageModel
 {
     private readonly ICategoryService _categoryService;
+    private readonly ILogger _logger;
 
     public IEnumerable<Category> Categories { get; private set; }
 
-    public CategoryModel(ICategoryService categoryService) => _categoryService = categoryService;
-
+    public CategoryModel(
+        ICategoryService categoryService,
+        ILogger<Category> logger)
+    {
+        _categoryService = categoryService;
+        _logger = logger;
+    }
     public async Task OnGet()
     {
+        _logger.LogInformation("Categories process start", 
+            DateTime.UtcNow.ToShortTimeString());
```

Comment:
Good usage!
You should also add log message that specifying time when method is finishing it's process work at the end of function.
Also, I advise you to add rows spaces between log calls for better code reading

Reply:
Don't forget to specify the finish date after 26 row)

Reply:
Upper comment is still relevant :)